setup.py

```diff
@@ -27,6 +27,10 @@ def run_tests(self):
 
 with open('requirements.txt', 'r') as f:
     required = f.read().splitlines()
+print("hacky thing for render-python dev branch. fix me.")
```

Comment:
Can we get rid of this now that render-python `2.2.1` is live?

Reply:
yes